routing_common/transit_types.cpp

```diff
@@ -74,6 +97,11 @@ SingleMwmSegment::SingleMwmSegment(FeatureId featureId, uint32_t segmentIdx, boo
 {
 }
 
+bool SingleMwmSegment::IsEqualForTesting(SingleMwmSegment const & s) const
```

Comment:
Why not to ```operator ==```?

Reply:
All transit data structure has method IsEqualForTesting(). It's use in a template for testing (routing_common_tests/transit_test.cpp/TestSerialiation()) and in case of double/PointD IsEqualForTesting() is not the same with operator==. Since instead of checking for equivalence it checks for almost equivalence.